Scripts/Tracking/CharacterController/CharacterControllerHeightSetter.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace VRTK.Core.Tracking.CharacterController
```

Comment:
What about the namespace of `Character` rather than `CharacterController`?

Then maybe the class could be `ControllerHeightSetter`?

so it's `Character.ControllerHeightSetter`?

Reply:
Actually with other classes below like `CharacterControllerMover` it then becomes a bit harder to understand what `ControllerMover` would do